doc/Caliptra.md

```diff
@@ -688,7 +688,7 @@ The following table describes the NIST SP 800-193 requirements that Caliptra sha
 | 4.1.1 | All security mechanisms and functions shall be founded to Roots of Trust (RoT). | Caliptra forms the basis for all trust in the SoC starting from execution of its immutable ROM. See the [Secure Boot Flow](#secure-boot-flow) section. |
 | 4.1.1 | If Chains of Trust (CoT) are used, RoT shall serve as the anchor for the CoT. | All other firmware shall be authenticated and executed as part of a Chain of Trust extended from the Caliptra ROM. See the [Secure Boot Flow](#secure-boot-flow) section. |
 | 4.1.1 | All RoTs and CoTs shall either be immutable or protected using mechanisms that ensure all RoTs and CoTs remain in a state of integrity. | All other firmware is authenticated and executed as part of a Chain of Trust extended from the Caliptra ROM. See the [Secure Boot Flow](#secure-boot-flow) section. |
```

Comment:
Suggesting the following:
> Caliptra firmware is authenticated and executed as part of a Chain of Trust extended from the Caliptra ROM. See the [Secure Boot Flow](#secure-boot-flow) section. The SoC or platform is responsible for maintaining integrity for other elements of the CoT.

Reply:
For lines 689 and 696, something like "All other firmware shall be measured into a CoT extended from the Caliptra ROM."

Reply:
done